keras/utils/dataset_utils.py

```diff
@@ -33,16 +33,16 @@
 def split_dataset(
     dataset, left_size=None, right_size=None, shuffle=False, seed=None
 ):
-    """Split a dataset into a left half and a right half (e.g. train / test).
+    """Split a dataset into a left part and a right part (e.g. train / test).
 
     Args:
         dataset: A `tf.data.Dataset` object, or a list/tuple of arrays with the
           same length.
-        left_size: If float (in the range `[0, 1]`), it signifies
+        left_size: If float (in the range "[0, 1] (excluding the boundary values 0 and 1)"), it signifies
```

Comment:
- use backticks for the `[]` range
- remove `"` quotes
- make sure the lines are under 80 chars